octobot_trading/exchanges/rest_exchange.py

```diff
@@ -364,10 +359,55 @@ def get_exchange_pair(self, pair: str) -> str:
         else:
             raise ValueError(f'{pair} is not supported')
 
+    def get_default_balance(self):
+        return self.client.account()
+
     def set_sandbox_mode(self, is_sandboxed):
         self.client.setSandboxMode(is_sandboxed)
 
     @staticmethod
-    def __get_side(order_type):
+    def _get_side(order_type):
         return TradeOrderSide.BUY.value if order_type in (TraderOrderType.BUY_LIMIT, TraderOrderType.BUY_MARKET) \
             else TradeOrderSide.SELL.value
+
+    """
+    Accounts
+    """
+    async def switch_to_account(self, account_type):
+        raise NotImplementedError("switch_to_account is not available on this exchange")
+
+    """
+    Parsers
+    """
+    def parse_balance(self, balance):
+        return self.client.parse_balance(balance)
+
+    def parse_trade(self, trade):
+        return self.client.parse_trade(trade)
+
+    def parse_order(self, order):
+        return self.client.parse_order(order)
+
+    def parse_ticker(self, ticker):
+        return self.client.parse_ticker(ticker)
+
+    def parse_ohlcv(self, ohlcv):
+        return self.client.parse_ohlcv(ohlcv)
+
+    def parse_order_book(self, order_book):
+        return self.client.parse_order_book(order_book)
+
+    def parse_timestamp(self, data_dict, timestamp_key):
+        return self.client.parse8601(self.client.safe_string(data_dict, timestamp_key))
+
+    def parse_currency(self, currency):
+        return self.client.safe_currency_code(currency)
+
+    def parse_status(self, status):
+        return OrderStatus(self.client.parse_order_status(status))
+
+    def parse_side(self, side):
+        return TradeOrderSide.BUY if side == self.BUY_STR else TradeOrderSide.SELL
+
+    def parse_account(self, account):
```

Comment:
👍 Shouldn't this method be called get_account though ?

Reply:
Not really, parsers are called after requesting exchange. They are normalizers. They fix and clean exchange specific dictionnaries to match Octobot format.